js/stores/windowStore.js

```diff
@@ -624,6 +625,10 @@ const doAction = (action) => {
           windowState = windowState.setIn(path.concat(['security', 'runInsecureContent']),
             action.securityState.runInsecureContent)
         }
+        if (action.securityState.evCert !== undefined) {
```

Comment:
i think this defaults to `null`, not `undefined`: https://github.com/brave/browser-laptop/pull/11776/files#diff-c3c556f1ee31674844d6d0be95748d8eR677

Reply:
nvm, i guess this has to check for `undefined` so `null` can be used to clear previous values